params/superchain.go

```diff
@@ -0,0 +1,96 @@
+package params
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+func init() {
+	for id, ch := range superchain.OPChains {
+		NetworkNames[fmt.Sprintf("%d", id)] = ch.Name
+	}
+}
+
+func OPStackChainIDByName(name string) (uint64, error) {
+	for id, ch := range superchain.OPChains {
+		if ch.Chain+"-"+ch.Superchain == name {
```

Comment:
So this winds up being `--beta.op-network=op-goerli` or `base-goerli` right?

Reply:
Yes indeed